pipeline/terraform/stack/service_ingestor_images.tf

```diff
@@ -26,7 +26,12 @@ module "ingestor_images" {
 
     es_images_index    = local.es_images_index
     es_augmented_index = local.es_images_augmented_index
-    es_is_reindexing   = var.reindexing_state.scale_up_tasks
+
+    # This var is read by the ingestor to switch the refresh interval off during a reindex
+    # It appears to be unnecessary, as the database copes well enough, so it is currently
+    # explicitly set to false, pending the removal of the switch.
+    # Was var.reindexing_state.scale_up_tasks
```

Comment:
Awesome, glad to see we can remove this. The ticket is https://github.com/wellcomecollection/catalogue-pipeline/issues/2238